### DISCUSSION TITLE
Escolha do Milestone block - Por quê Muir Glacier?

### DISCUSSION THREAD

**Discussion by alehoss** [bacen/pilotord-kit-onboarding#28] [Q&A] [answered]
Olá pessoal,

Seguindo a [documentação do Hyperledger Besu](https://besu.hyperledger.org/stable/public-networks/reference/genesis-items#milestone-blocks), recomenda-se utilizar a release mais recente, que nesse caso seria Shangai

> In private networks, we recommend specifying the latest milestone block. It's implied this includes the preceding milestones. This ensures you use the most up-to-date protocol and have access to the most recent opcodes.

Também da mesma doc:

> Ensure you include a milestone far enough in advance in the genesis file. Not doing so can lead to unexpected and inconsistent behavior without specific errors.

Sendo assim, gostaria de entender o racional por trás da escolha do milestone block ser [Muir Glacier](https://github.com/bacen/pilotord-kit-onboarding/blob/main/genesis.json#L4) . Obrigado 🙇‍♂️

**Comment by aldenio** [comment, marked as answer]
Prezado @alehoss,

Sim, você está certo de que Besu tem suporte para o milestone Xangai desde a versão 23.1.2 para cadeias mainnet/públicas, porém, segundo sua equipe de desenvolvimento há uma limitação atual para redes permissionadas (IBFT/QBFT).

O milestone escolhido pelo time, tem nos atendido plenamente. A maioria das atualizações recentes no protocolo estão relacionadas à mudança de PoW para PoS e não são tão relevantes para redes permissionadas. Estamos aguardando o suporte completo ao milestone Xangai para atualizarmos nossa rede adicionando suporte ao novo (e mais eficiente) opcode "PUSH0".

Agradecemos sua contribuição e nos colocamos a disposição para novos esclarecimentos.
